extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/error/CancellationReason.java

```diff
@@ -43,11 +45,24 @@ public enum CancelationReason
 
   private final String reason;
 
-  CancelationReason(String reason)
+  CancellationReason(String reason)
   {
     this.reason = reason;
   }
 
+  @JsonCreator
+  public static CancellationReason fromString(String value)
+  {
+    for (CancellationReason mode : values()) {
+      if (mode.toString().equals(value)) {
```

Comment:
This won't work, because it uses `toString` to deserialize, but Jackson's default serializer for enum uses `name()`. It's better to use the name here, in case we decide to change the string message. So try replacing this with `mode.name().equals(value)`

Reply:
I noticed that tests were failing and just pushed this fix!